### PR TITLE
Create Style Component

### PR DESCRIPTION
**Pull request by victormaestri**
# PR Details

## Description

Create a new style component to image for just add new style `max-width: 100%`. This style become image more responsive.

## Steps to Reproduce
If you toggle your chrome window for mobile device the image will break.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/containers/Home/index.jsx [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { withTranslation } from 'react-i18next'
 import logo from 'assets/images/juggernaut.gif'
-import { StyledContainer, StyledTitle, StyledSubtitle } from './style'
+import { StyledContainer, StyledTitle, StyledSubtitle, StyleImage } from './style'
 
 const Home = ({ t }) => (
   <StyledContainer>
-    <img
+    <StyleImage
```
Comment (marlonmleite) [inline src/containers/Home/index.jsx:9]: Prefixar conforme os demais componentes com `Styled`

Comment (victormaestri) [reply src/containers/Home/index.jsx:9]: Sorry, errei ao nomear o style component

Comment (victormaestri) [reply src/containers/Home/index.jsx]: Done

**Review by vilaboim — APPROVED** [review]

**Review by marlonmleite — APPROVED** [review]

**Comment by vilaboim** [comment]
@all-contributors please add @victormaestri for code

**Comment by vilaboim** [comment]
@all-contributors please add @victormaestri for code

**Pull request merged**